Sprint-3/implement/rotate-char.js

```diff
@@ -41,3 +41,46 @@ console.log(rotateCharacter("7", 5)); // Output: "7" (unchanged, not a letter)
 // And the function should return the rotated character as a string (e.g., 'z' rotated by 3 should become 'c', 'Z' rotated by 3 should become 'C').
 console.log(rotateCharacter("z", 1)); // Output: "a" (preserves case, but wraps around)
 console.log(rotateCharacter("Y", 2)); // Output: "A" (preserves case, but wraps around)
+
+function rotateCharacter(char, n) {
+  if (!/^[a-zA-Z]$/.test(char)) return "Unchanged, not a letter";
```

Comment:
I think you're adding in quite a few extra conditions here. Could you simplify this code a bit? How would you go about doing that?

Reply:
Thank you for the feedback.
I’ve simplified the code by reducing the number of conditions and using regular expressions to check if the character is a letter (uppercase or lowercase).